braviarc/braviarc.py

```diff
@@ -231,6 +231,10 @@ def set_volume_level(self, volume):
     def turn_on(self):
         """Turn the media player on."""
         self._wakeonlan()
+        # Try using the power on command incase the WOL doesn't work
+        POWER_ON_CMD = 'AAAAAQAAAAEAAAAuAw=='
```

Comment:
When I was looking for the command using this link http://openremote.org/display/forums/Sony+TV+HTTP+control it looked like the command is being used being used as the mute command. But I found somewhere else that this same command actually switches on the tv.
I have tested it using X80C and X85 and it is working on both